src/components/settings/SettingsLMSTab/index.jsx

```diff
@@ -62,7 +61,21 @@ const SettingsLMSTab = ({
     setShowNoConfigCard(false);
     // Since the user is editing, hide the existing config cards
     setConfigsExist(false);
-  };
+    openLmsStepper();
+  }, [dispatch, openLmsStepper]);
```

Comment:
We aren't using `openLmsStepper`in this callback anymore, right?
```suggestion
  }, [dispatch, configData]);
```

Reply:
Open lms stepper is still being used, see line 64

Reply:
Gotcha, not sure how I missed that...